pubsub/drivertest/drivertest.go

```diff
@@ -94,6 +99,10 @@ func (verifyAsFailsOnNil) SubscriptionCheck(s *pubsub.Subscription) error {
 	return nil
 }
 
+func (verifyAsFailsOnNil) ErrorCheck(t *pubsub.Topic, err error) error {
+	return nil
```

Comment:
This should call `t.ErrorAs(nil)` and verify it returns false (like the other tests in the struct.

Reply:
Replaced with a panic check.